endpoint_route_handler/models/endpoint_route_handler.py

```diff
@@ -323,3 +305,35 @@ def _get_routing_info(self):
             csrf=self.csrf,
         )
         return route, routing, self.endpoint_hash
+
+    def write(self, vals):
+        if any([x in vals for x in self._controller_fields() + ("active",)]):
```

Comment:
Why isn't "active" returned by `_controller_fields` ? This is a field that requires a sync, right ?

Reply:
those fields are only the ones that are used for the real route and to generate the hash. `active` shouldn't fit

Reply:
I already renamed it to `_routing_impacting_fields` anyway ;P